packages/paste-core/components/form-pill-group/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,121 @@
+import * as React from 'react';
+import {render} from '@testing-library/react';
+import {matchers} from 'jest-emotion';
+import {CustomizationProvider} from '@twilio-paste/customization';
+// @ts-ignore typescript doesn't like js imports
+import axe from '../../../../../.jest/axe-helper';
+import {useFormPillState, FormPillGroup, FormPill} from '../src';
+import {Basic, SelectAndRemove, CustomFormPillGroup} from '../stories/index.stories';
+
+expect.extend(matchers);
+
+const CustomElementFormPillGroup: React.FC = () => {
+  const pillState = useFormPillState();
+
+  return (
+    <form>
+      <FormPillGroup
+        {...pillState}
+        element="CUSTOM_PILL_GROUP"
+        data-testid="form-pill-group"
+        aria-label="Your favorite sports:"
+      >
+        <FormPill {...pillState} element="CUSTOM_PILL" data-testid="form-pill">
+          Tennis
+        </FormPill>
+      </FormPillGroup>
+    </form>
+  );
+};
+
+describe('FormPillGroup', () => {
+  // Verifies that the correct aria attributes and semantics are met
```

Comment:
Nit: Instead of comments here, could this information be included in the `describe` and/or `it` blocks? These comments provide helpful context, which I think could be super useful to see in the test suite command line output.